src/main/java/com/stripe/model/EphemeralKey.java

```diff
@@ -43,24 +67,21 @@ public static EphemeralKey create(Map<String, Object> params, RequestOptions opt
     return request(RequestMethod.POST, classUrl(EphemeralKey.class), params, EphemeralKey.class,
         options);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="delete">
   /**
-   * Delete an ephemeral key.
+   * Invalidates a short-lived API key for a given resource.
    */
   public EphemeralKey delete() throws StripeException {
     return delete(null);
   }
 
   /**
-   * Delete an ephemeral key.
+   * Invalidates a short-lived API key for a given resource.
```

Comment:
Minor but I have a preference for keeping "ephemeral key" since it's the object name. No strong feels though

Reply:
Fixed.